app/HutchManager/Dto/QueryResult.cs

```diff
@@ -0,0 +1,37 @@
+using System.Text.Json.Serialization;
+
+namespace HutchManager.Dto
+{
+  public class QueryResult
+  {
+    public string Status { get; set; } = "OK";
+    public string ProtocolVersion { get; set; } = "2";
+
+    [JsonPropertyName("activity_source_id")]
+    public int ActivitySourceId { get; set; }
+
+    [JsonPropertyName("job_id")] public string JobId { get; set; } = String.Empty;
+
+    [JsonPropertyName("queryResult")]
+    public QueryResultCount Results
+    {
+      get { return _result; }
+      set
+      {
+        _result = value;
+        _count = value.Count;
+      }
+    }
+
+    private QueryResultCount _result;
```

Comment:
⛏️ Could this be done to suppress the build warning?
```suggestion
    private QueryResultCount? _result;
```

Reply:
I'm wondering if it's best to set a default value instead? We don't really want QueryResultCount to be allowed to be null

Reply:
Good point. I suppose if something goes wrong we just say the result count is 0, right?